workflows/pipe-common/scripts/autoscale_sge.py

```diff
@@ -2201,6 +2355,18 @@ def main():
     scale_down_timeout = int(api.retrieve_preference('ge.autoscaling.scale.down.timeout', default_value=30))
     scale_up_polling_timeout = int(api.retrieve_preference('ge.autoscaling.scale.up.polling.timeout',
                                                            default_value=900))
+    common_utils = ScaleCommonUtils(api=api)
+    static_host_storage = FileSystemHostStorage(cmd_executor=cmd_executor,
+                                                storage_file=os.path.join(work_dir,
+                                                                          '.static.%s.storage' % queue_name),
+                                                clock=clock)
+    run_activation_timeout = 30
+    if static_hosts_number:
```

Comment:
Grid engine autoscaler manages static hosts only if the current queue is static. If queue is static then static hosts are part of this queue. Therefore, we should you the following if here:

```suggestion
    if queue_static and static_hosts_number:
```